pkgs/build-support/rust/hooks/maturin-build-hook.sh

```diff
@@ -34,6 +32,9 @@ maturinBuildHook() {
     mkdir -p dist
     mv target/wheels/*.whl dist/
 
+    # These are python build hooks and may depend on ./dist
+    runHook postBuild
+
```

Comment:
One example: [python-relax-deps-hook](https://github.com/NixOS/nixpkgs/blob/9e17672672a9906a9f313601c6301d4ac7f95580/pkgs/development/interpreters/python/hooks/python-relax-deps-hook.sh#L78), which is a  `postBuild` hook.